### PR TITLE
repacle means "替换‘’ or “代替” in Chinese #document

### PR DESCRIPTION
**Pull request by Yuntao-Wu**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

...

Detailed description / Documentation draft:

...

By adding documentation, you'll allow users to try your new feature immediately, not when someone else will have time to document it later. Documentation is necessary for all features that affect user experience in any way. You can add brief documentation draft above, or add documentation right into your patch as Markdown files in [docs](https://github.com/ClickHouse/ClickHouse/tree/master/docs) folder.

If you are doing this for the first time, it's recommended to read the lightweight [Contributing to ClickHouse Documentation](https://github.com/ClickHouse/ClickHouse/tree/master/docs/README.md) guide first.

### CODE REVIEW

**Comment by blinkov** [comment]
cc @zhang2014 @sundy-li

File: docs/zh/engines/table-engines/mergetree-family/replacingmergetree.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# 更换麦树 {#replacingmergetree}
+# 替换合并树 {#replacingmergetree}
```
Comment (zhang2014) [inline docs/zh/engines/table-engines/mergetree-family/replacingmergetree.md:1]: Maybe nouns should not be translated? ReplacingMergeTree is better?

File: docs/zh/engines/table-engines/mergetree-family/replacingmergetree.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ CREATE TABLE [IF NOT EXISTS] [db.]table_name [ON CLUSTER cluster]
 
 请求参数的描述，参考[请求参数](../../../engines/table-engines/mergetree-family/replacingmergetree.md)。
 
-**替换树参数**
+**替换合并树参数**
```
Comment (zhang2014) [inline docs/zh/engines/table-engines/mergetree-family/replacingmergetree.md:26]: 替换合并树参数 -> 参数？

File: docs/zh/engines/table-engines/mergetree-family/replacingmergetree.md (hunk 1 above)
Comment (Coder-taotao) [reply docs/zh/engines/table-engines/mergetree-family/replacingmergetree.md:1]: It sounds like better. but I can't tolerate the wrong translated version.

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**